package.json

```diff
@@ -11,7 +11,7 @@
     "e2e": "npm run swagger && ng e2e",
     "clean": "rimraf ./src/swagger-data && rimraf ./documentation",
     "swagger": "java -jar ./swagger/swagger-codegen-cli.jar generate -i ./swagger/swagger.json -l typescript-angular2 -t ./swagger -o ./src/swagger-data",
-    "example-data": "node ./server/index.js",
+    "example-data": "ts-node ./server/index.ts",
```

Comment:
Since this requires the swagger schema to run, we may want to run the swagger command before serving the files.  Similar to `npm start`